licence_inserter.js

```diff
@@ -0,0 +1,47 @@
+const replace = require('replace-in-file');
+const fs = require('fs');
+const { resolve } = require('path');
```

Comment:
what does this syntax do differently?

Reply:
I don't really know, I copied it from an example. It may depend on how old the library is. The way importing works seems to have changed over time